cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -202,14 +203,19 @@ private Future<Void> initialReconcile(VertxTestContext context) {
                 StrimziPodSet sps = supplier.strimziPodSetOperator.client().inNamespace(namespace).withName(KafkaResources.kafkaComponentName(CLUSTER_NAME)).get();
                 assertThat(sps, is(notNullValue()));
 
+                X509Certificate serverCert = Ca.x509Certificate(MockCertManager.serverCert().getBytes());
+
                 sps.getSpec().getPods().stream().map(PodSetUtils::mapToPod).forEach(pod -> {
                     assertThat(pod.getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLIENTS_CA_CERT_GENERATION, "0"));
                     assertThat(pod.getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, "0"));
                     assertThat(pod.getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_KEY_GENERATION, "0"));
                     var brokersSecret = client.secrets().inNamespace(namespace).withName(KafkaResources.kafkaSecretName(CLUSTER_NAME)).get();
+                    var brokersSecretKey = Ca.SecretEntry.CRT.asKey(pod.getMetadata().getName());
+
                     assertThat(pod.getMetadata().getAnnotations(), hasEntry(Annotations.ANNO_STRIMZI_SERVER_CERT_HASH,
-                            CertUtils.getCertificateThumbprint(brokersSecret, Ca.SecretEntry.CRT.asKey(pod.getMetadata().getName()))
+                            CertUtils.getCertificateThumbprint(brokersSecret, brokersSecretKey)
                     ));
+                    assertThat(serverCert.getNotAfter().getTime(), is(CertUtils.getCertificateExpirationDateEpoch(brokersSecret, brokersSecretKey)));
```

Comment:
What does this actually test? Is it intended to test `CertUtils.getCertificateExpirationDateEpoch`? Would that be better placed in some `CertUtilsTest` class? This way it will get easily lost in some future test refatoring.

Reply:
Same as the arg above regarding `getCertificateExpirationDateEpoch` was putting it where `getCertificateThumbprint` and `getCertificateShortThumbprint` where tested 😄

Reply:
I disagree. The other asserts here test that the rcorrect value is added as an annotation to the pod. But you do not seem to test it, you seem to essentially just test that the `getCertificateExpirationDateEpoch` returns the expected value.

Reply:
Let me move it to a new `CaTest`